.yarn/versions/38fca589.yml

```diff
@@ -0,0 +1,2 @@
+releases:
+  "@yarnpkg/plugin-dlx": prerelease
```

Comment:
You'll need to mark the CLI for release too; the others can be set to "decline".